packages/shared/src/contexts/BootProvider.tsx

```diff
@@ -54,13 +58,20 @@ function filteredProps<T extends Record<string, unknown>>(
 export type BootDataProviderProps = {
   children?: ReactNode;
   app: string;
+  localBootData?: BootCacheData;
   getRedirectUri: () => string;
 };
 
-const getLocalBootData = () => {
+export const getLocalBootData = (isInitialLoad = false): BootCacheData => {
```

Comment:
This function should not apply the theme but only return what is stored in the cache

Reply:
Makes total sense. Let me work it out.